lib/MusicBrainz/Server/Entity/Editor.pm

```diff
@@ -342,6 +340,8 @@ sub TO_JSON {
         entityType => 'editor',
         gravatar => $self->gravatar,
         id => $self->id,
+        is_bot => boolean_to_json($self->is_bot),
+        is_limited => boolean_to_json($self->is_limited),
```

Comment:
Is there any reason why these should be only in UnsanitizedEditor? It's public info on the site and it feels wrong to pass around an unsanitized editor just for them.

Reply:
We now output `privileges`, which you can use for `is_bot`, at least. I don't see a problem with outputting `is_limited`.

Reply:
Make sure to also update the "Unsanitized editor data was found on the client" check, though.